build/teamcity/cockroach/nightlies/compose.sh

```diff
@@ -6,6 +6,12 @@ dir="$(dirname $(dirname $(dirname $(dirname "${0}"))))"
 source "$dir/teamcity-support.sh"
 source "$dir/teamcity-bazel-support.sh"
 
+# The test failures generated by TestComposeCompare are not necessarily
+# failures per se. They're cases of behavioral divergences from Postgres. While
+# our compatibility guarantees are not 100%, it's better to treat failures as
+# information to occasionally review.
+export SKIP_LABEL_TEST_FAILURE=1
```

Comment:
is this the way to do it, as opposed to how the env vars are passed on line 34? i guess the difference is if the variable is used by the test runner versus the test itself?

Reply:
Yes, this is correct. We want the variable to be visible by `bazci`. Compare to line 34 which sets environment variables for the test inside the sandbox.